app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -87,6 +87,8 @@ define(function (require, exports, module) {
             avatarWrapperEl
               .removeClass('with-default')
               .append($(profileImage.get('img')).addClass('profile-image'));
+            $('#image-holder').css('display','inline-block');
```

Comment:
add a space after `,` in both cases `.css('display','inline-block');` -> `.css('display', 'inline-block');

Reply:
OK